src/main/java/domain/LottoStore.java

```diff
@@ -0,0 +1,51 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class LottoStore {
+    static private final int LOTTO_LIMIT_LENGTH = 6;
```

Comment:
구글 스타일 가이드를 보시면 접근제어자가 우선 오게 됩니다.
(https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)

```suggestion
    private static final int LOTTO_LIMIT_LENGTH = 6;
```